eventq_aws/lib/eventq_aws/aws_queue_worker.rb

```diff
@@ -77,7 +77,8 @@ def start_process(options, queue, block)
         @thread_count.times do
           thr = Thread.new do
 
-            client = options[:client]
+            client = options[:client] || new_client_instance(options) # singleton or non-singleton
```

Comment:
Ok, then just put a comment here as we really don't want to support this.  And maybe a TODO to remove potentially in a major release.